src/widget/wmainmenubar.cpp

```diff
@@ -60,6 +60,11 @@ WMainMenuBar::WMainMenuBar(QWidget* pParent, UserSettingsPointer pConfig,
 }
 
 void WMainMenuBar::initialize() {
+    WMainMenuBar* target = this;
+    createMenu([target](QMenu* x) { target->addMenu(x); });
+}
+
+void WMainMenuBar::createMenu(std::function<void(QMenu*)> pAddMenu) {
```

Comment:
Renaming suggestion (not yet part of the coding guidelines):
`pAddMenu` -> `fnAddMenu`